Lockbox.xcodeproj/project.pbxproj

```diff
@@ -1333,7 +1333,7 @@
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				CODE_SIGN_STYLE = Automatic;
 				DEFINES_MODULE = YES;
-				DEVELOPMENT_TEAM = 43AQ936H96;
```

Comment:
It's totally fine to change these bits of the `.pbxproj` file for developing / deploying locally, but you will need to take them out of the PR before I can accept it.